components/chip/overview.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Overview
+page_title: Chip Overview
+description: Overview of the Chip for Blazor.
+slug: chip-overview
+tags: telerik,blazor,chip,overview
+published: True
+position: 0
+---
+
+# Blazor Chip Overview
+
+The <a href="https://www.telerik.com/blazor-ui/chip" target="_blank">Blazor Chip component</a> shows a piece of information in a compact form. The chip can be selected, removed or disabled. You can respond to various user interactions through the exposed [events]({%slug chip-events%}), and customize the [appearance]({%slug chip-appearance%}) of the Telerik Chip for Blazor.
+
+## Creating Blazor Chip
+
+To use a Telerik Chip for Blazor:
+
+1. Add the `TelerikChip` tag
+1. Set the `Selected` parameter to a `boolean` (supports one-way and two-way binding).
+1. (Optional) Set the `Text` parameter to a render a string in the chip. 
```

Comment:
Perhaps the optional approach is to use ChildContent, while `Text` should be the default approach?